libraries/Microsoft.Bot.Builder.Runtime/Builders/IBuilder.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Bot.Builder.Runtime.Builders
+{
+    public interface IBuilder<out TItem>
```

Comment:
Style comment, I would use T instead of TItem (normally you use TItem when you have many Ts but we use T when there's just one).